hunting/llm/queries/aws_bedrock_ignore_previous_prompt_detection.toml

```diff
@@ -0,0 +1,32 @@
+[hunt]
+author = "Elastic"
+description = "This analytic flags multiple instances of LLM vulnerability to malicious user interaction, helping to maintain ethical guidelines and compliance standards."
```

Comment:
I would consider adding a bit more specificity here around the "malicious user interaction" behavior being captured, since your rule really is specifically targeting malicious attempts to have the LLM ignore previous prompts.

Reply:
Done.